dev/com.ibm.ws.microprofile.opentracing.jaeger/src/com/ibm/ws/microprofile/opentracing/jaeger/JaegerTracerFactory.java

```diff
@@ -327,4 +338,28 @@ private static String resolveValue(String value) {
         return value;
     }
 
+    private static boolean isBeta() {
+        try {
+            final Map<String, ProductInfo> productInfos = ProductInfo.getAllProductInfo();
+
+            for (ProductInfo info : productInfos.values()) {
+                if (EARLY_ACCESS.equals(info.getEdition())) {
+                    return true;
+                }
+            }
+        } catch (Exception e) {
+            Tr.debug(tc, "Exception getting InstalledProductInfo: " + e.getMessage());
+            e.printStackTrace();
+        }
+        return false;
+    }
+    
+    private static boolean isEnabledByUser() {
+        Boolean isEnabled  = getBooleanProperty(ENV_JAEGER_ENABLEMENT);
```

Comment:
Why do you need this environment variable?

Reply:
This is used to enable Jaeger in OL builds as OL doesn't have beta builds.

Reply:
No but it does have development builds ... https://openliberty.io/downloads/#development_builds

Those contain all features, not just the GA ones. What does ProductInfo.getEdition() return for those?

Reply:
Development builds have same properties as a GA build.  These are what I got from one.

```
com.ibm.websphere.productId=io.openliberty
com.ibm.websphere.productOwner=Open Liberty Project
com.ibm.websphere.productVersion=19.0.0.11
com.ibm.websphere.productName=Open Liberty
com.ibm.websphere.productInstallType=Archive
com.ibm.websphere.productEdition=Open
com.ibm.websphere.productLicenseType=EPL
```

Reply:
OK, fair enough